docker/nginx/redirects-301.map

```diff
@@ -3,6 +3,7 @@
 ~^/service-docs/(.*)$ /services/$1;
 ~^/1.11/networking/edge-lb/(.*)$ /services/edge-lb/0.1.9/$1;
```

Comment:
Need to change all the other links to point to the `1.0.0` documentation as well.